guides/ruby_on_rails_quick_start.mdx

```diff
@@ -0,0 +1,125 @@
+---
+title: Ruby on Rails quick start — Meilisearch documentation
+description: Integrate Meilisearch into your Ruby on Rails app.
+---
+
+# Ruby on Rails quick start
```

Comment:
I recommend making it clear from the start this guide is focussed on Meilisearch Cloud. Don't need to make a big fuss out of it, just mention it early on.

Reply:
It also works with self-hosted Meilisearch. What do you think of adding the following sentence in step 1?

`If you prefer to use the self-hosted version of Meilisearch, you can follow the[ quick start](https://www.meilisearch.com/docs/learn/getting_started/quick_start) tutorial.`

Reply:
Sounds good to me!